dissect/target/exceptions.py

```diff
@@ -114,3 +114,7 @@ class RegistryCorruptError(RegistryError):
 
 class ConfigurationParsingError(Error):
     """An error occurred during configuration parsing."""
+
+
+class CRCMismatchException(Error):
+    """A mismatch between CRC checksums has occurred."""
```

Comment:
I'd move this out of `dissect.target.exceptions` and into your plugin. Maybe even get rid of it and just raise a `PluginError` with an appropriate message.

Reply:
This Exception is no longer used, so I'll just move it back to the shimcache plugin